backstop/scenarios/progress-indicator.js

```diff
@@ -4,11 +4,11 @@ module.exports = [
   // Progress indicator types
   scenario('linear', {
     selectors: ['#demo-linear'],
-    hideSelectors: ['.iui-indeterminate'],
+    hideSelectors: ['.iui-indeterminate', 'h3'],
```

Comment:
Maybe better change placement of `<section>`. I see in other tests we wrap with section only test elements and headers are outside.

Reply:
The red boxed in area is the `<section>`.  I did place the `<h2>` outside, consistent with other tests, but I left the `<h3>` inside and hid them.  The alternative would be to have each section under an `<h3>` within their own `<section>` but that would introduce more test images.
![Screen Shot 2021-05-20 at 11 01 44 AM](https://user-images.githubusercontent.com/849817/119002327-a2300180-b941-11eb-84f6-c1a00ef11df4.png)